unit_tests/speech/_fixtures.py

```diff
@@ -0,0 +1,12 @@
+SYNC_RECOGNIZE_RESPONSE = {
```

Comment:
1. Can you put the `Copyright` header in this file as well?
1. Is this file really needed? This variable is tiny and could just as easily be put inline wherever it is used.